src/prototype_room_creepbuilder.js

```diff
@@ -225,6 +226,7 @@ Room.prototype.getSettings = function(creep) {
       }
       setting = setting[foundKey];
     }
+    room.log(settingName, setting);
```

Comment:
we need it but only as debug... may be better a `config.debug.spawn && this.log(settingName, ' : ', 'setting');`.
about settingName, it shouldn't be easy to read without :
the methodgetSettings have as goal to take the settings setup in role files and 'filter' them for only keep the right ones.
e.g:
```
amount : {
  1: [2,3],
  5: [4,6]
},
...
```
will only return a final amount for creep bodyparts of [2,3], [4,6] or undefined.
settingName will be amount and setting the value to keep and this for each settings.